cmd/dagger/llm.go

```diff
@@ -97,6 +97,7 @@ func (s *LLMSession) reset() {
 	s.llm = s.dag.LLM(dagger.LLMOpts{Model: s.model}).
 		WithEnv(s.dag.Env(dagger.EnvOpts{
 			Privileged: true,
+			Module:     s.dag.CurrentModule().Source().AsModule(),
```

Comment:
this is clearly the wrong approach for passing the module as initialSelection, but i'm not sure what'd be better... CurrentModule seems to not be set in the shell case, and it's pretty weird going CurrentModule.Source.AsModule anyways.

can we figure out the client's -m loaded module from the server side and just pass a bool here? Maybe we should just be passing a generic Object?